pennylane/math/multi_dispatch.py

```diff
@@ -932,11 +932,13 @@ def jax_argnums_to_tape_trainable(qnode, argnums, expand_fn, args, kwargs):
     """
     import jax
 
-    with jax.core.new_main(jax.ad.JVPTrace) as main:
-        trace = jax.ad.JVPTrace(main, 0)
+    with jax.core.new_main(jax.interpreters.ad.JVPTrace) as main:
+        trace = jax.interpreters.ad.JVPTrace(main, 0)

```

Comment:
Out of curiosity, does this limit support for older JAX versions? E.g., when was `ad` first moved to the `interpreters` module?

No need to take any action here, especially because our aim is to only support latest ML versions :) But could be useful knowledge to have if we notice users having issues.

Reply:
I tried looking for that, but could not find the answer. There is no mention of its introduction in the changelog.

Reply:
Maybe here? https://github.com/google/jax/commit/365262b77a45bcde66058c6921a8d2771af460f2